components/EntryList/EntryList.js

```diff
@@ -1,58 +1,53 @@
-import { nanoid } from "nanoid";
-import { StyledEntryList } from "./EntryList.styled";
+import Animation from "../Animation/Animation";
 import Intensity from "@/utils/intensity";
+import { useState, useEffect } from "react";
+import * as Styled from "./EntryList.styled";
 
-import {
-  StyledEntry,
-  SliderText,
-  ExperienceText,
-  EntryText,
-  Separator,
-  EditDeleteButton,
-  StyledEntryHeadnote,
-} from "@/components/EntryList/EntryList.styled";
+export default function EntriesList() {
+  const [dataFromLocalStorage, setDataFromLocalStorage] = useState([]);
 
-export default function EntryList({ allEntries, deleteEntry }) {
-  function handleDeleteEntryListItem(id) {
-    deleteEntry(id);
+  useEffect(() => {
+    const dataFromLocalStorage = localStorage.getItem("anonymous_moods");
+    if (dataFromLocalStorage) {
+      const parsedData = JSON.parse(dataFromLocalStorage);
+      setDataFromLocalStorage(parsedData);
+    }
+  }, []);
```

Comment:
there is no reason why you should use a useEffect here. 
I really recommend you use the LocalStorageState Hook here, it makes life so much easier!